sdk/maps/maps-route-rest/README.md

```diff
@@ -0,0 +1,190 @@
+
+#### Using an Azure Active Directory Credential
+
+You can authenticate with Azure Active Directory using the [Azure Identity library][azure_identity]. To use the [DefaultAzureCredential][defaultazurecredential] provider shown below, or other credential providers provided with the Azure SDK, please install the `@azure/identity` package:
+
+```bash
+npm install @azure/identity
+```
+
+You will also need to register a new AAD application and grant access to Azure Maps by assigning the suitable role to your service principal. Please refer to the [Manage authentication](https://docs.microsoft.com/azure/azure-maps/how-to-manage-authentication) page.
+
+Set the values of the client ID, tenant ID, and client secret of the AAD application as environment variables: `AZURE_CLIENT_ID`, `AZURE_TENANT_ID`, `AZURE_CLIENT_SECRET`.
+
+You will also need to specify the Azure Maps resource you intend to use by specifying the `clientId` in the client options.
+The Azure Maps resource client id can be found in the Authentication sections in the Azure Maps resource. Please refer to the [documentation](https://docs.microsoft.com/azure/azure-maps/how-to-manage-authentication#view-authentication-details) on how to find it.
+
+```javascript
+const { createMapsRouteClient } = require("@azure-rest/maps-route");
+const { DefaultAzureCredential } = require("@azure/identity");
+
+const credential = new DefaultAzureCredential();
+const client = createMapsRouteClient(credential, "<maps-account-client-id>");
+```
+
+#### Using a Subscription Key Credential
+
+You can authenticate with your Azure Maps Subscription Key.
+
+```javascript
+const { createMapsRouteClient } = require("@azure-rest/maps-route");
+const { AzureKeyCredential } = require("@azure-core-auth");
+
+const credential = new AzureKeyCredential("<subscription-key>");
+const client = createMapsRouteClient(credential);
+```
+
+## Examples
+
+The following sections provide several code snippets covering some of the most common Azure Maps Route tasks, including:
+
+- [Request historic and real-time data](#request-historic-and-real-time-data)
+- [Request a route for a commercial vehicle](#request-a-route-for-a-commercial-vehicle)
+- [Calculate and optimize a multi-stop route](#calculate-and-optimize-a-multi-stop-route)
+
+### Request historic and real-time data
+
+By default, the Route service assumes the traveling mode is a car and the departure time is now. It returns route based on real-time traffic conditions unless a route calculation request specifies otherwise.
+
+To retrieve the route direction, you need to pass in the parameters the coordinates through which the route is calculated, delimited by a colon. A minimum of two coordinates is required. The first one is the origin and the last is the destination of the route.
```

Comment:
```suggestion
To retrieve the route direction, you need to pass in the parameters the coordinates through which the route is calculated, delimited by a colon. A minimum of two coordinates are required. The first one is the origin and the last is the destination of the route.
```

Reply:
Thanks for pointing this out! Fixed in 96781dc3df277391148097eb60c21688978091e5